Scripts/lint.sh

```
@@ -27,54 +27,44 @@ else
 	exit
 elif [ "$LINT_MODE" = "STRICT" ]; then
 	SWIFTFORMAT_OPTIONS="--configuration .swift-format"
 	SWIFTLINT_OPTIONS="--strict"
+	STRINGSLINT_OPTIONS="--config .strict.stringslint.yml"
 else
 	SWIFTFORMAT_OPTIONS="--configuration .swift-format"
 	SWIFTLINT_OPTIONS=""
+	STRINGSLINT_OPTIONS="--config .stringslint.yml"
 fi
 
 pushd $PACKAGE_DIR
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)